docs/howto/django.rst

```diff
@@ -0,0 +1,40 @@
+Use Procrastinate in a Django application
+=========================================
+
+Install procrastinate with:
+
+.. code-block:: console
+
+    $ (venv) pip install 'procrastinate[django]'
+
+This tells pip to consider the extra dependencies from the group of dependencies named
+`django`. For now, this group only contains Django itself, which you likely already
+have in your project's dependencies. So why bother ?
+
+Specifying your dependency to the "`django` extras" will ensure that your Django version
```

Comment:
Do you really want to use single back-ticks around `django` here?

Reply:
Ah I meant double backticks.